src/generic/loading-spinner/index.tsx

```diff
@@ -4,7 +4,7 @@ import { useIntl } from '@edx/frontend-platform/i18n';
 import messages from './messages';
 import { LoadingSpinnerProps } from './types';
 
-export const LoadingSpinner = ({ size = 'lg' }: LoadingSpinnerProps) => {
+export const LoadingSpinner = ({ size }: LoadingSpinnerProps) => {
```

Comment:
It seems that the `lg` size is not supported by the Spinner component.

Bootstrap only provides the `sm` size option: https://react-bootstrap-v4.netlify.app/components/spinners/

<img width="1047" height="322" alt="Image" src="https://github.com/user-attachments/assets/24dfe918-46d3-44c5-a16a-5d88ababbac2" />

Reply:
[question] Does it make sense to pass the `size` parameter if it can only be of the `sm` value you defined below?